src/tokens/transformation/android/formats/dimenStyle.js

```diff
@@ -0,0 +1,20 @@
+const { fileHeader } = require('style-dictionary').formatHelpers
+const changeCase = require('change-case')

```

Comment:
It doesn't look like we depend on this package. Ideally, we'd avoid depending on it at all